### ISSUE TITLE
Long Wave in Center Edit (was: Snake Oil Reel)

### ISSUE THREAD

**Issue by dcmorse** [contradb/contra#366]
http://contradb.com/dances/312
http://lists.sharedweight.net/pipermail/callers-sharedweight.net/2009-June/002056.html

A1, beats 9-12: should be "ladles dance out" not "ladles dance in"
A2: need a second balance, need to say "neighbors allemande right 1/2" (new wave format will help)

Pasting in from the sharedweight archive linked above:

#### A2
- Men allemande Left ¾ (4); take right hands with neighbor to form a wavy line across the set, and **balance(4)**
- Allemande right neighbor ½, to form wavy line across, with women in center (4), and **balance (4)**

**Comment by cranhandler** [comment]
I came across this in the wave audit, and thanks for covering all the bases on the 'long wave' in and outs. 

BUT---I want this second one to say "gents dance out while ladles dance in to a long wave in the center - balance the wave" , and it currently says "ladies dance out while gents dance in to a long wave in the center - balance the wave"

Can we flip that sentence so the new "dancer set" who needs to switch from 'waiting' to 'dancing' gets their name said first? 

![image](https://user-images.githubusercontent.com/16180173/40285664-80a5ced8-5c64-11e8-8323-dc359ddda719.png)

**Comment by dcmorse** [comment]
Have you considered that the old way led to fewer subject switches when you read it combined with the previous figure?:

* Ladies dance in and ladies balance the wave.
* Ladies dance out while...
* Gents dance in and gents balance the wave.

Total number of changes in subject: 1

Versus this proposed way:
* Ladies dance in and ladies balance the wave
* Gents dance in while....
* ladies dance out and gents balance the wave.

Total number of changes in subject: 3

Totally willing to go either way, but this is a common combination of figures, and that's why I made it the way I did.
